foundry.toml

```diff
@@ -8,8 +8,13 @@ build_info = true
 ffi = true
 ast = true
 extra_output = ["storageLayout"]
+evm_version = "cancun"
+
```

Comment:
What does this serve and what evm version were we using before 😅? The latest?

Reply:
Need it to compile the Curve interface to the correct bytecode to interact with the forked pool.